cockatrice/src/player.cpp

```diff
@@ -2612,6 +2612,8 @@ void Player::updateCardMenu(const CardItem *card)
 
         if (card->getZone()) {
             if (card->getZone()->getName() == "table") {
+                // Card is on the battlefield
```

Comment:
I'm not sure this comment adds any clarity; the line above says the same thing.

Reply:
Added since I added to all of them